extensions/wikia/WikiaHubsServices/models/EditHubModel.class.php

```diff
@@ -299,6 +302,9 @@ public function getModuleUrl( $timestamp, $moduleId ) {
 		);
 	}
 
+	/**
+	 * @return SpecialPage
```

Comment:
http://stackoverflow.com/a/3121559/1050577

It's basically the same as `new $className()`